### DISCUSSION TITLE
6772708892466f8c6ec39e9e

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#14] [General]
# 6772708892466f8c6ec39e9e

Discover a wide range of cracked games, mods, and gaming resources for PC, Mac, and more. Download your favorite games for free and enjoy endless gaming adventures!

https://toxicgames.in/6772708892466f8c6ec39e9e

**Comment by Admin-ToxicGames** [comment]
<h2>Change Language</h2>

Change the language of the game, right click on the game, show package contents, go to the "Contents" folder, run Wineskin, change the value to "Windows app":

"C:\Program Files (x86)\Mass Effect 2 LE\Game\ME2\Binaries\Win64\MassEffect2.exe" -NoHomeDir -SeekFreeLoadingPCConsole -Subtitles 20 -OVERRIDELANGUAGE=RUS

On:

"C:\Program Files (x86)\Mass Effect 2 LE\Game\ME2\Binaries\Win64\MassEffect2.exe" -NoHomeDir -SeekFreeLoadingPCConsole -Subtitles 20 -OVERRIDELANGUAGE=ENG

Click Test Run, the run will be in English.
